docs/en/authenticators.rst

```diff
@@ -263,6 +263,7 @@ Configuration options:
    ``null`` and all pages will be checked.
 -  **passwordHasher**: Password hasher to use for token hashing. Default
    is ``DefaultPasswordHasher::class``.
+-  **salt**: When `true`, the application's Security salt will be appended to the token before hashing. When `false`, no salt is used. When a string is passed this will be used as a salt instead. Default is `true`.
```

Comment:
```suggestion
-  **salt**: When ``true``, the application's Security salt will be appended to the token before hashing. When ``false``, no salt is used. When a string is passed this will be used as a salt instead. Default is ``true``.
```